src/View/Printing/PaymentReminder/PaymentReminderView.php

```diff
@@ -35,7 +51,8 @@ public function getFileName(): string
 
     public function getSubject(): string
     {
-        return __('Payment Reminder') . ' '
+        return $this->model->reminder_level . ' '
```

Comment:
is this intended ex. "3 Payment Reminder Order 12345"

Reply:
yes